packages/block-library/src/image/index.js

```diff
@@ -76,6 +70,19 @@ const blockAttributes = {
 	height: {
 		type: 'number',
 	},
+	fileWidth: {
+		type: 'number',
+	},
+	fileHeight: {
+		type: 'number',
+	},
+	userSet: {
```

Comment:
While in the context of the pull request the name doesn't draw much attention, but without that as context, it'd be hard to associate this being related to dimensions of the image. I'm thinking maybe a suffix of "Dimensions" ?

Reply:
Was trying to come up with the shortest names that make sense as these will be repeating a lot in all posts. Feel free to change as you like :)